docs/semgrep-appsec-platform/jira.md

```diff
@@ -15,10 +15,17 @@ The Semgrep Jira integration allows you to create Jira tickets based on your Sem
 
 ## Prerequisites
 
-* This feature is in a **private beta**. To request access:
-  1. Fill out the [Request access to the Semgrep Jira integration closed beta](https://get.semgrep.dev/Jira-asana-linear-private-beta.html) form.
-  2. Contact your Technical Account Manager or Account Executive and let them know you want to try the Jira integration.
-* You must have a **Jira Cloud** plan. Jira Data Center (self-managed or on-premise) is not supported.
+- The Jira integration is in **public beta** for all existing Semgrep AppSec Platform users.
+- You must have a **Jira Cloud** plan. Jira Data Center (self-managed or on-premise) is not supported.
```

Comment:
@jsmoxon Please let me know if this is still up-to-date for June 26? Thank you!

Reply:
Yes this is correct for June 26th. GA will be a few weeks after based on final feedback.